specifyweb/settings/specify_settings.py

```
@@ -59,6 +59,17 @@
 # database. This can be the same as the Specify 6 master user.
 MASTER_NAME = 'MasterUser'
 MASTER_PASSWORD = 'MasterPassword'
+MIGRATOR_NAME = 'MasterUser'
+MIGRATOR_PASSWORD = 'MasterPassword'
```

Ruff (0.15.14)

[error] Possible hardcoded password assigned to: "MIGRATOR_PASSWORD"

(S105)